docs/_templates/topbar/launchbuttons.html

```diff
@@ -30,21 +33,6 @@
         <a class="dropdown-buttons"
             href="https://scipp.github.io/release/0.8.4"><button type="button"
                 class="btn btn-secondary topbarbtn">v0.8</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.7.1"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.7</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.6.1"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.6</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.5.0"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.5</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.4.0"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.4</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.3.0"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.3</button></a>
```

Comment:
Why did you remove these?

Reply:
I felt the menu was getting too long to be useful. No one is using these anymore, so other than support a historian I see little use?

Reply:
Then what is the point of keeping the actual pages around? Should we decide generally on when we remove links / pages?

Reply:
- Yes we should probably (at some point) also remove the pages. They take a lot of space.
- I don't think if we can determine a fixed cycle at this point. "3 years" might work once we are quite stable, but now that is useless.